presto-druid/pom.xml

```diff
@@ -155,6 +155,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is this related to the upgrade?

Reply:
Yes - - the org.testcontainers 1.14.3 jar depended on org.jetbrains.annotations, but the 1.15.0-rc2 jar does not.